aioftp/server.py

```diff
@@ -1389,7 +1389,7 @@ async def _start_passive_server(self, connection, handler_callback):
                     )
                     connection.passive_server_port = port
                     break
-                except asyncio.QueueEmpty:
+                except asyncio.QueueEmpty as e:
```

Comment:
What this solves?

Reply:
without the from, nothing. I should have removed it.
```suggestion
                except asyncio.QueueEmpty:
```